.github/workflows/ci_cd.yml

```diff
@@ -668,6 +668,86 @@ jobs:
         env:
           DEPLOYMENT_ENV: production
 
+  nuxt-load-test:
```

Comment:
Maybe the load testing could be split to a separate action?

Reply:
Potentially, and it's something I plan to look into when adding the job to run k6 against staging in the next PR after this one. The steps to run k6 on a local environment are so different than running it against a remote environment, I wanted to make sure I didn't abstract too early :slightly_smiling_face: